insert.js

```diff
@@ -0,0 +1,20 @@
+var mysql = require('mysql');
```

Comment:
What is this file being used for?

Reply:
This is needed for retrieving the database. Also, for the connection.

Reply:
I'm talking about the file `insert.js`. Is the file really needed?

Reply:
we don't need it. I was testing on it. I will delete it.